.github/workflows/auto-updater.yaml

```diff
@@ -0,0 +1,29 @@
+---
+name: Validate Auto Updater
+on:
+  push:
+    paths:
+      - 'downloads/hashes.txt.sig'
+  pull_request:
+    paths:
+      - 'downloads/hashes.txt.sig'
+jobs:
+  validate-auto-updater:
+    name: Validate Auto Updater
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v1
```

Comment:
v3 is available. Should we switch to that one?

Reply:
I intentionally use v1, there is no real advantage for us with v2 or v3